packages/p2p/src/stores/my-ads-store.js

```diff
@@ -519,13 +539,30 @@ export default class MyAdsStore extends BaseStore {
         this.show_edit_ad_form = show_edit_ad_form;
     }
 
+    @action.bound
+    setIsSwitchModalOpen(is_switch_modal_open, app_id) {
+        this.setSelectedAdId(app_id);
+        this.is_switch_modal_open = is_switch_modal_open;
+    }
+
+    @action.bound
+    setShouldSwitchAdRate(is_switch_ad_rate) {
+        this.is_switch_ad_rate = is_switch_ad_rate;
```

Comment:
This function is confusing because it has `should` in the name but the value it sets is called `is_switch_ad_rate`.  The value and the setter should match and be called accordingly.

Reply:
Will rename it

Reply:
We normally follow this practise for setters name `is_example => setIsExample`. Also, I think it makes more sense to keep `should` inside so you maybe can rename it to `setShouldSwitchAdRateStatus` and the variable to `should_switch_ad_rate`

Reply:
Will rename the function and variable